cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -400,101 +426,108 @@ SwiftGenerator::writeOpDocSummary(IceInternal::Output& out, const OperationPtr&
         }
     }
 
-    const ParamInfoList allInParams = getAllInParams(p);
     auto docParameters = doc->parameters();
-    for (ParamInfoList::const_iterator q = allInParams.begin(); q != allInParams.end(); ++q)
+
+    // Document all the in parameters.
```

Comment:
If there would be **more than** 2 parameters, use the list style instead of the line style.
```
- Parameter hello: a
- Parameter there: b
```
vs
```
- Parameters:
  - hello: a
  - there: b
```